README.md

```diff
@@ -24,170 +24,73 @@
+Running Nocturne on your Car Thing requires a host device such as a Raspberry Pi, Windows PC, Mac, or Linux PC, as well as a Spotify Premium account. You will also need [superbird-tool](https://github.com/thinglabsoss/superbird-tool) to flash the image to the CarThing regardless of your computer's operating system. Alternatively, there is a web version of superbird-tool called [Terbium](https://terbium.app/).
 
-### Windows
+### Flashing the Spotify CarThing
 
-#### Raspberry Pi Setup
-
-Download and open [Raspberry Pi Imager](https://downloads.raspberrypi.org/imager/imager_latest.exe), select Raspberry Pi OS (Legacy, 64-bit) Lite, select "Edit Settings", check "Set hostname", check "Set username and password" (set a password), check "Configure wireless LAN", (enter your network's SSID and password), check "Set local settings". Open the Services tab, enable SSH, and use password authentication. Write the configured OS to your microSD card and insert it into your Raspberry Pi.
-
-#### Flashing Process
-
-If you haven't already, download [superbird-tool](https://github.com/thinglabsoss/superbird-tool) and run the setup process detailed [here](https://github.com/thinglabsoss/superbird-tool?tab=readme-ov-file#supported-platforms).
-
-Download and unzip the latest image from [Releases](https://github.com/brandonsaldan/nocturne-image/releases), connect Car Thing to your computer in USB Mode (hold preset buttons 1 and 4 while connecting), and run the following from your command line:
-
-```bash
-# Go into the superbird-tool repository
-$ cd C:\path\to\superbird-tool-main
-
-# Find device
-$ python superbird_tool.py --find_device
-
-# Flash Nocturne image, without resetting the data partition 
-$ python superbird_tool.py --dont_reset --restore_device C:\path\to\nocturne-image\image 
-```
-After the flashing completes, connect your Raspberry Pi to your computer, and change directories to the scripts folder.
-
-```bash
-# Go into the setup-scripts folder
-$ cd \path\to\nocturne-image\setup-scripts
-```
-
-There are two different ways to use Nocturne. You can either use it at your desk, or in your car. 
+Download and unzip the latest image from [Releases](https://github.com/brandonsaldan/nocturne-image/releases), connect Car Thing to your computer in USB Mode (hold preset buttons 1 and 4 while connecting). Then follow either of the below flashing methods.
 
 <details>
-<summary>Using Nocturne at your desk</summary>
-<br>
-Connect your Car Thing to the Raspberry Pi and run the following from your command line:
-
-```bash
-# Transfer setup_host_rpi.sh to Raspberry Pi
-$ scp \path\to\nocturne-image\setup-scripts\setup_host_rpi.sh pi@raspberrypi.local:/home/pi/
+<summary>Using Terbium (Recommended)</summary>
 
-# SSH into Raspberry Pi
-$ ssh pi@raspberrypi.local
+<br>
```

Comment:
These manual line breaks add an unnatural spacing, I think they should be removed.